app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is used to store `Person` instances by name. This approach assumes that each person has a unique name. If names are not unique, this could lead to overwriting existing entries. Consider using a different key or structure to ensure uniqueness.